<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>860</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lab_work_4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл з Х значеннями</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл з Y значеннями</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_x_but">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>79</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Обрати</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_x">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>input_x.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_y">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>input_y.txt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_y_but">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Обрати</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>451</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип поліному</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cheb_first">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Чебишева першого порядку</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cheb_sec">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Чебишева другого порядку</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lag">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Лаггера</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ermit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ерміта</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lej">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Лежандра</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Степінь поліному</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Х1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x1_grade">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Х2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Х3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x2_grade">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="x3_grade">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Координата у</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_cord">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="normalize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Нормалізувати</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Обчислити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_graph">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Побудувати графіки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вибірка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Налаштування моделі та поліному</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Модель</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Адитивна</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mult">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Мультиплікативна</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
